doc/architecture/decisions/0010-use-beds-instead-of-spaces-for-matching.md

```diff
@@ -0,0 +1,23 @@
+# 10. Use Beds instead of Spaces for matching
+
+Date: 2023-03-29
+
+## Status
+
+Accepted
+
+## Context
+
+There are different approaches that can be made to match people into AP's and the approach chosen will
+the UI, API, and business processes. Generally it comes down to matching people into beds or spaces within an AP.
```

Comment:
I think this first sentence is missing a word '...the approach chosen will effect the UI...' maybe?

Reply:
Good catch